lib/concurrent/atomic/condition.rb

```diff
@@ -66,5 +66,17 @@ def broadcast
       true
     end
 
+    private
+
+    if defined?(Process::CLOCK_MONOTONIC)
+      def clock_time
+        Process.clock_time Process::CLOCK_MONOTONIC
```

Comment:
Yes it should be.  That's what I get for not running the tests. >_<

I'll update the patch